astropy/modeling/fitting.py

```diff
@@ -1656,7 +1656,7 @@ def _fitter_to_model_params(model, fps):
                 model._array_to_parameters()
 
 
-def _model_to_fit_params(model):
+def model_to_fit_params(model):
```

Comment:
Since those "private" functions are most probably used in external code, we should keep them for a while with a deprecation warning.

Reply:
What version of astropy should we deprecate these in?

Reply:
5.1?

Reply:
Yes, next major release so 5.1.

Reply:
See commit b866d17.